kornia/feature/loftr/loftr.py

```diff
@@ -67,24 +88,33 @@ def __init__(self, config=default_cfg, pretrained: Optional[str] = None):
         self.fine_preprocess = FinePreprocess(config)
         self.loftr_fine = LocalFeatureTransformer(config["fine"])
         self.fine_matching = FineMatching()
-
+        self.pretrained = pretrained
         if pretrained is not None:
             if pretrained not in urls.keys():
                 raise ValueError(f"pretrained should be None or one of {urls.keys()}")
             pretrained_dict = torch.hub.load_state_dict_from_url(
-                 urls[pretrained], map_location=lambda storage, loc: storage)
+                urls[pretrained], map_location=lambda storage, loc: storage)
             self.load_state_dict(pretrained_dict['state_dict'])
+        self.eval()
 
-    def forward(self, data):
+    def forward(self, data: Dict) -> Dict:
         """
-        Update:
+        Args:
             data (dict): {
```

Comment:
```suggestion
            data: {
```
not sure how will render the one below